product/dashboard/widgets/cpu_right_sizing_recommendations.yml

```diff
@@ -0,0 +1,23 @@
+description: CPU Right Sizing Recommendations
```

Comment:
The extension should be `yaml` and not `yml`.

Reply:
Fixed.